CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
 #============================================================================
 # Initialize the project
 #============================================================================
-project(ignition-utils0 VERSION 0.1.0)
+project(ignition-utils VERSION 1.0.0)
```

Comment:
this also drops the major version from the project name. I think this is intentional, but I want to confirm

cc @j-rivero

Reply:
Good point, `ign-plugin` does have the 1. Adding it back

Reply:
So, to clarify, are we putting `1` in the debian package name as well? I see the homebrew package does have the version number.

Reply:
I'm following `ign-plugin`'s pattern:

* homebrew: 1
* deb: no number
* cmake: 1